src/content/docs/setup/openid.mdoc

```diff
@@ -118,11 +119,13 @@ auth:
         clientid: <vikunja client-id>
         clientsecret: <vikunja client-secret>
         scope: openid profile email
+        forceuserinfo: false  # Optional: Set to true to always use UserInfo endpoint instead of ID token claims
```

Comment:
```suggestion
        forceuserinfo: false # Optional: Set to true to always use UserInfo endpoint instead of ID token claims, defaults to false
```